src/main/resources/application.properties

```diff
@@ -5,7 +5,9 @@ spring.datasource.tomcat.validation-query=SELECT 1
 spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
 spring.liquibase.enabled=false
 spring.profiles.active=google
-springfox.documentation.swagger.v2.path=/api-docs
+spring.resources.cache.cachecontrol.max-age=0
+spring.resources.cache.cachecontrol.must-revalidate=true
```

Comment:
Can you explain why these are needed?

Reply:
If I understand correctly, this is because we're changing how we host the swagger ui html.  This lines up with the caching rules in other Terra services